front-end/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false
```

Comment:
Is there a way we can go about not disabling this? I think it could open up a few future bugs considering the number of devs we currently have. These bugs could either make our app less stable or harder to debug, since I don't know if we can catch all of them in the PRs

Reply:
Talked with Chris offline, simply changing .then and .catch callback parameters to (res: any) and (err: any) prevents me from having to disable this TS config lint. Changes are now in.